omod/src/main/webapp/datasets/datasetEditor.jsp

```diff
@@ -38,16 +37,46 @@
 				$j('#fixedValue${p.field.name}').show();
 			}
 		</c:forEach>
-
+		
 	} );
+	
+	$j(document).ready(function() {
```

Comment:
We don't need 2 ready functions in the same page, do we?  Can't we just add this functionality within the existing ready function?